modulefiles/modulefile.ProdGSI.jet

```diff
@@ -35,10 +34,12 @@ module load w3emc-intel-sandybridge/2.2.0
 module load w3nco-intel-sandybridge/2.0.6
 module load sigio-intel-sandybridge/2.0.1
 module load sfcio-intel-sandybridge/1.0.0
+module load ip-intel-sandybridge/3.0.0
 
-module use /home/George.Vandenberghe/t1/l1025/lib/modulefiles
-module load hdf5
-module load netcdf
+#module use /home/George.Vandenberghe/t1/l1025/lib/modulefiles
+#module load hdf5
+#module load netcdf
 
-export NETCDF=${NETCDF4}
-export NETCDF_DIR=${NETCDF4}
+module load netcdfp/4.7.4
+#export NETCDF=${NETCDF4}
```

Comment:
and these commented out export commands, please remove

Reply:
Removed.